### DISCUSSION TITLE
two way SSL with rocket

### DISCUSSION THREAD

**Discussion by vu3mmg** [rwf2/Rocket#2056] [General]
Hi,
Could you please point me to examples with two way SSL with rocket .
-mahesh

**Comment by somehowchris** [comment]
May you mean mTLS?

**Reply by vu3mmg** [reply]
Both client and server authenticates each other.
Server will trust client's certificate and client trust servers .
Yes Mutual TLS .

**Reply by somehowchris** [reply]
https://api.rocket.rs/master/rocket/mtls/index.html

**Reply by somehowchris** [reply]
You need to point your rocket dependency to the master branch of the repository to have it available

**Comment by vu3mmg** [comment]
Thank you  for the help  .

**Reply by somehowchris** [reply]
Did it answer your question? If so please mark this thread as answered

**Reply by vu3mmg** [reply]
I am not able to  find the tick mark for marking it as answered
<img width="911" alt="Screenshot 2022-01-10 at 6 59 24 AM" src="https://user-images.githubusercontent.com/9333815/148709369-30052233-d45a-4c01-8795-42f3c10397bb.png">
d
